assets/javascripts/discourse-assign/routes/group-assigned.js.es6

```diff
@@ -14,6 +14,10 @@ export default DiscourseRoute.extend({
       group: this.modelFor("group"),
     });
     controller.group.set("assignment_count", model.assignment_count);
+    controller.group.set(
```

Comment:
Let's combine this line with the line above by using `setProperties` to update multiple attributes.